webapp/src/App.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div id="app" class="apollo">
+    <div id="grettings">
+      {{ message }}
+    </div>
+    <div v-if="user != null">
+      <TodoList :user="this.user" />
+    </div>
+    <div v-else>
+      <Login @set-user="setUser" />
+    </div>
+  </div>
+</template>
+
+<script>
+import gql from "graphql-tag";
+import TodoList from "./components/TodoList";
+import Login from "./components/Login";
+
+export default {
+  name: "App",
+  components: {
+    Login,
+    TodoList
+  },
+  apollo: {
+    message: gql`
+      query {
+        message: hello
+      }
+    `
+  },
+  data() {
+    return {
+      message: "Hello Vue!",
+      isLoggedIn: false,
+      user: null
+    };
+  },
+  methods: {
+    setUser(user) {
+      this.user = user;
+      localStorage.setItem("token", user.token);
```

Comment:
I recommend the [login helpers](https://github.com/nuxt-community/apollo-module#authentication) that come with `apollo` module for [NuxtJS](https://nuxtjs.org/).

Reply:
I am not going to implant NuxtJs